src/java.desktop/windows/classes/sun/awt/Win32GraphicsDevice.java

```diff
@@ -441,6 +443,18 @@ public synchronized void setFullScreenWindow(Window w) {
     protected native void enterFullScreenExclusive(int screen, WindowPeer w);
     protected native void exitFullScreenExclusive(int screen, WindowPeer w);
 
+    /**
+     * Reapplies the size of this device to the full-screen window.
```

Comment:
What is _this device_? There's no device in the parameters of the method.

If you write javadoc, you should document the parameters too.

Reply:
@aivanov-jdk I've updated the JavaDoc to more specifically reference the GraphicsDevice that owns the added method (and documented the parameters)

Reply:
Ah, I overlooked the fact that it's a method of a `GraphicsDevice`.